include/ledRunner.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __LED_RUNNER_H
+#define __LED_RUNNER_H
+
+#include <FastLED.h>
+
+typedef enum {
+    LED_MODE__OFF = 0,
+    LED_MODE__SOLID,
+    LED_MODE__BLINK,
+    LED_MODE__BREATH,
+    LED_MODE__COUNTDOWN,
+    LED_MODE__NUMMODES
+} ledMode_t;
+
+#endif /* __LED_RUNNER_H */
```

Comment:
Placeholder -- will eventually have a class that will drive a giver player's LEDs according to any number of differnt modes